src/test/java/integration/EsContainer.java

```diff
@@ -0,0 +1,19 @@
+package integration;
+
+
+import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.utility.DockerImageName;
+
+public class EsContainer extends ElasticsearchContainer {
+
+    public EsContainer() {
+        super(DockerImageName.parse("elasticsearch:8.5.3")
+                .asCompatibleSubstituteFor(
+                        "docker.elastic.co/elasticsearch/elasticsearch")
+        );
+        this.addFixedExposedPort(9200, 9200);
```

Comment:
@hizmailovich ctors must not contain any code

read [this](https://www.yegor256.com/2015/05/07/ctors-must-be-code-free.html)